app/frontend/courses/review/components/CoursesReview__ChecklistShow.res

```diff
@@ -118,6 +147,9 @@ let make = (
   let (selection, setSelecton) = React.useState(() => [])
   let (id, _setId) = React.useState(() => DateTime.randomId() ++ "-review-checkbox-")
 
+  let (addAdditionalFeedback, setAddAdditionalFeedback) = React.useState(() =>
```

Comment:
The name of the state variable could just be _additonalFeedback_ . The `add` prefix makes it sound like an action. Update the handler function name accordingly.